specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -4014,19 +4683,57 @@
             "modelAsString": true
           }
         }
-      },
-      "discriminator": "computeType"
+      },
+      "discriminator": "computeType"
+    },
+    "PaginatedServiceList": {
```

Comment:
do we need to define this?

Reply:
I did not define this. I dont know why this was an addition here.

Reply:
Down below, it assumes that I changed the PaginatedServiceList line for PaginatedInferenceDeploymentsList